### PR TITLE
update prod settings image

### PR DESCRIPTION
**Pull request by MitchellBarker**
## What are you changing in this pull request and why?
We've changed the UI for checking production in an env. This is because we're very soon releasing a 'staging' environment as well.

Could be better, but we're releasing the new staging type soon, so this will be fine temporarily imo:
Screenshot 2023-12-11 at 1 08 49 PM

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

**Review by matthewshaver — APPROVED** [review]

**Comment by matthewshaver** [comment]
Thank you @MitchellBarker ! Is this good to go live now or in the future?

**Comment by MitchellBarker** [comment]
@matthewshaver It's good as of the release today!

**Review by thorn14 — APPROVED** [review]

**Comment by MitchellBarker** [comment]
To be clear, I can't merge this, so @matthewshaver if you could 🙏 
Screenshot 2023-12-14 at 9 32 28 AM

**Pull request merged**
